rectools/visuals/metrics_app.py

```diff
@@ -124,7 +155,7 @@ def fold_ids(self) -> tp.List[int]:
         """Sorted list of fold identifiers from the `models_metrics`."""
         return sorted(self.models_metrics[Columns.Split].unique())
 
-    def _validate_input_data(self) -> None:
+    def _validate_models_metrics(self) -> None:
         if not isinstance(self.models_metrics, pd.DataFrame):
             raise ValueError("Incorrect input type. `metrics_data` should be a DataFrame")
         if Columns.Split not in self.models_metrics.columns:
```

Comment:
Can we add a possibility for user to pass models_metrics without a split column? Assuming that there is only one fold.
Can we do it in this PR or should we do it afterwards?

Reply:
Nice idea, should be easy to implement. I think we could add in in this PR.

Reply:
Done